spec/i18n_spec.rb

```diff
@@ -174,14 +174,6 @@ def allowed_untranslated_key?(locale, key)
             bad_keys = keys.reject { |key| key =~ /^[a-z0-9_.]+$/ }
             expect(bad_keys).to be_empty
           end
-
-          it 'has only has XML-safe identifiers (keys start with a letter)' do
-            keys = flattened_yaml_data.keys
-
-            bad_keys = keys.select { |key| key.split('.').any? { |part| part =~ /^[0-9]/ } }
-
-            expect(bad_keys).to be_empty
-          end
```

Comment:
Wow what a throwback, I forgot why we had this.... old i18n export format 6de798511b244a241151e79613cf004ea535f89d